### PR TITLE
Bump version to Omni Core 0.0.12.0-rel

### PR DESCRIPTION
**Pull request by dexX7**
This bumps the version to 0.0.12.0 and prepares the release notes.

### CODE REVIEW

**Comment by dexX7** [comment]
@zathras-crypto: I started with filling out the blanks here: 0fc1775a64ac1f24cc488ee88d08dd3d5d9e9b25.

I think we can get away with recycling the descriptions for the fee system and cross-property STO, instead of mentioning that we just fixed some stuff. What do you think?

Most of the release notes can probably also be used for 0.2.

**Comment by dexX7** [comment]
@zathras-crypto: what do you think about the release notes? Please let me know, if there is anything to change, or add. :) Feel free to type as you like..

**Comment by zathras-crypto** [comment]
Hey bud, this all looks good to me - though I'm not sure we're making it clear enough that this is a mandatory upgrade.  For example:

> This version is built on top of v0.0.11, which is a major release and consensus critical in terms of the Omni Layer protocol rules. An upgrade is mandatory, and highly recommended. Prior releases will not be compatible with new behavior in this release.

I read that as 0.0.11 is the major release and required upgrade, which may give the impression that this isn't mandatory.  Since activation will specify 0.0.11.3 as a min client version for fees & cross-prop STO, clients that haven't been upgraded will stop working.

TL:DR; I think we should be a bit clearer that this version is required once we start activating things :)

**Comment by dexX7** [comment]
Maybe we should call it 0.0.12, to have at least some consistency, in terms of bumping the version number for consensus critical stuff?

**Comment by zathras-crypto** [comment]
Sounds perfect, I don't think we have to worry about version confusion with Bitcoin Core now - 0.0.12 would be fine :)

**Comment by dexX7** [comment]
Here we go: https://github.com/OmniLayer/omnicore/releases/tag/v0.0.12

**Comment by dirvine** [comment]
To compile that you need to let users know to set CXXFLAGS with fPIC, it needs exported as passing 
`CXXFLAGS=-fPIC ./configure` will not be enough
